src/test/java/org/hiero/sketch/TableTest.java

```diff
@@ -6,10 +6,33 @@
 
 import static org.hiero.sketch.DoubleArrayTest.generateDoubleArray;
 import static org.hiero.sketch.IntArrayTest.generateIntArray;
-import static org.junit.Assert.assertEquals;
+import static org.hiero.sketch.IntArrayTest.getRandIntArray;
```

Comment:
We don't like static imports; please use just import and in the code say IntArrayTest.getRandIntArray().